manage.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python
+  
 import os
 import sys
 
 if __name__ == '__main__':
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'gitmate.settings')
 
+   
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.python.analysis`.*

The issue can be fixed by applying the following patch:

```diff
--- a/manage.py
+++ b/manage.py
@@ -6,7 +6,7 @@
 if __name__ == '__main__':
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'gitmate.settings')
 
-   
+
     from django.core.management import execute_from_command_line
 
     execute_from_command_line(sys.argv)
```